### PR TITLE
[TASK] Add test-docs to make file

### PR DESCRIPTION
**Pull request by linawolf**
Also adjust the gitignore as the newest ddev versions do not create one automatically anymore

releases: main, 12.4, 11.5

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ nbproject
 Thumbs.db
 var/
 .cache/
+/.ddev/*
```
Comment (brotkrueml) [inline .gitignore:36]: Actually, .ddev should bring its own .gitignore file, so these entries should not be necessary.

Comment (brotkrueml) [reply .gitignore:36]: I suggest to decouple this change from the Makefile.

Comment (linawolf) [reply .gitignore:36]: ddev in the newest versions does not bring its own ignore file anymore. and there is really no harm in excluding this.

Comment (brotkrueml) [reply .gitignore:36]: See #458

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**